src/app/emote-menu/SubtemberPromotion.vue

```diff
@@ -2,13 +2,10 @@
 	<div class="seventv-promotion-subtember">
 		<div>
 			<p class="header">Get exclusive cosmetics and emotes!</p>
-			<p class="description">
-				Gift atleast three 7TV subs to unlock exclusive cosmetics only aquirable during this event
-			</p>
+			<p class="description">Gift 3+ 7TV subs to unlock exclusive cosmetics for a limited time</p>
```

Comment:
Should probably be the other way around, 

`For a limited time, Gift 3+ 7TV subs to unlock exclusive cosmetics`

Reply:
Having it at the end implies that the cosmetics will not be usable after the event is over

Reply:
oh right yeah